### PR TITLE
build: update to rules_nodejs 1.2.2

### PR DESCRIPTION
**Pull request by gregmagolan**
This brings in a fix so that yarn_install picks up the yarn-path attribute of the .yarnrc file.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by josephperrott — APPROVED** [review]
LGTM

**Comment by gregmagolan** [comment]
Waiting https://github.com/bazelbuild/rules_nodejs/pull/1582 and a rules_nodejs 1.2.1 release after that lands

**Comment by gregmagolan** [comment]
Windows CI turned on here to make sure this doesn't break it: https://github.com/angular/angular/pull/35061

**Review by josephperrott — APPROVED** [review]

File: WORKSPACE [hunk 1]
```diff
@@ -29,7 +29,7 @@ Try running `yarn bazel` instead.
     minimum_bazel_version = "1.1.0",
 )
 
-check_rules_nodejs_version(minimum_version_string = "1.0.0")
+check_rules_nodejs_version(minimum_version_string = "1.2.0")
```
Comment (josephperrott) [inline WORKSPACE]: This should be `1.2.1` correct?

File: WORKSPACE [hunk 2]
```diff
@@ -29,7 +29,7 @@ Try running `yarn bazel` instead.
     minimum_bazel_version = "1.1.0",
```
Comment (josephperrott) [inline WORKSPACE]: Can you set this to be `2.0.0`?

**Review by josephperrott — CHANGES_REQUESTED** [review]

File: WORKSPACE (hunk 1 above)
Comment (gregmagolan) [reply WORKSPACE]: 👍 Yes. 1.2.0 would break the build.

File: WORKSPACE (hunk 2 above)
Comment (gregmagolan) [reply WORKSPACE]: 👍

**Review by josephperrott — APPROVED** [review]

**Pull request closed without merging**